WIP: add ci with travis

dlemireanybox opened the pull request:
Please do not  merge

Commit:
#9685: start working with buildout

Commit:
README: add travis status

Commit:
travis: fix upgrade scritp name

Commit:
Add module coop_product_coefficient into upgrade_lalouve  #9685

petrus-v commented:
You can see travis build here https://travis-ci.org/dlemireanybox/odoo-foodcoops/builds/

In the future you will have to configure travis to allow it to access to that repo to test 9.0 branch and pull requests

petrus-v commented:
@simonjejcic

For your information I've test pushing to coveralls.io, it needs some authorization as well to let travis push coverall information, you can [see a non relevant result](here https://coveralls.io/github/petrus-v/odoo-foodcoops ) (as the db test is actually not ready for tests) on one of my repo branch. [here a python file from la louve module result](https://coveralls.io/builds/13700016/source?filename=louve_addons%2Fpurchase_package_qty%2Fmodel%2Fstock_picking.py)

All this will needs [more configuration](http://coveralls-python.readthedocs.io/en/latest/tips/coveragerc.html) to ignore some directories. But this is actually a great tool to see what part of code is test or not.

 @dlemireanybox could you add some tips how to build the environment and launch unit test on a local dev machine into the README.md.

Add a link to [the documentation to prepare the environment for buildout](http://docs.anybox.fr/anybox.recipe.odoo/current/first_steps.html)  and assume he is ready to create a virtual python environement!

Commit:
#FIX werzeug 0.9.6 version into buildout
#FIX upgrade problem caused by pos_product_barcodes,
pos_automatic_cashdrawer

petrus-v commented on intercoop_addons/pos_product_barcodes/__openerp__.py:
```diff
@@ -13,7 +13,9 @@
-        'barcodes'
+        'barcodes', 
+        'stock',
+        'point_of_sale'
```
Just for information, to get a nice git diff you can add the coma after ``'point_of_sale',`` so next time you add a dependency at the end of the list you only get the diff on the additional module. I know this is perfectionism!

petrus-v commented on upgrade_lalouve.py:
```diff
@@ -0,0 +1,117 @@
+        'stock_expense_transfer',
+        # en dessous de la ligne install 1 dépendance
+        'account_bank_statement_import_caisse_epargne',
+'account_bank_statement_import_ofx',
```
Could you use the same indent, flake8 would complains about that! (it doesn't because we are not testing this file: https://travis-ci.org/dlemireanybox/odoo-foodcoops/builds/289931019#L4293)

Commit:
[FIX]upgrade for pos_product_barcodes and do not install
coop_product_coefficient without FIX
product.template.get_auto_update_theorical_cost (safe_eval),
product.template.get_auto_update_theorical_price (safe_eval) and Taxes
15% with wrong setting

petrus-v commented:
Replace by #381
